### ISSUE TITLE
refactor rust syntax and features and release pdf version

### ISSUE THREAD

**Issue by leopeng1995** [carols10cents/language-to-rust-cheat-sheets#4]
I just searched [go-lang-cheet-sheet](https://github.com/a8m/go-lang-cheat-sheet) and reviewed a lot key concepts from it. I appreciated that work, and I also be interested in Rust. So I found your awesome project. And I think your project will be popular in some day.

So can we refactor this project to similar form releasing pdf version or something?

**Comment by carols10cents** [comment]
A PDF version would be neat! It would just be a matter of putting in the time. I'm not sure when I'll have time to work on this in the near future, if you would like to help, that would be great!

**Comment by leopeng1995** [comment]
@carols10cents Yep, I am glad to have a chance to contribute this project during my Rust learning time.

From now on, the TODO list contains:
- [ ] convert existed document to pdf version
- [ ] add more languages supported

Do you agree?

**Comment by carols10cents** [comment]
Yup sounds good!
